packages/react-notion-x/src/components/asset.tsx

```diff
@@ -3,7 +3,7 @@ import { BaseContentBlock, Block } from 'notion-types'
 import { getTextContent } from 'notion-utils'
 
 import { useNotionContext } from '../context'
-import { LazyImage } from './lazy-image'
+import NextImage from 'next/image'
```

Comment:
We will likely need to add `image` to our overridable props s.t. the client application inserts `next/image` instead of the library itself. This will allow us to not take `next` as a dependency.

Reply:
I removed `next` as dependency! 695690f